lib/dishes.rb

```diff
@@ -0,0 +1,25 @@
+class Dishes
```

Comment:
If this class is making a single dish, it might be clearer for a reader if you call it Dish instead of Dishes - because Dishes makes it sound like this class might represent a menu.